### ISSUE TITLE
IMPROVE: Cart Page CSS

### ISSUE THREAD

**Issue by KendallDoesCoding** [anubhav06/eatit#21]
Does the cart page use something close to the Swiggy Template?

![image](https://user-images.githubusercontent.com/84712013/154982230-683829ce-7664-4337-92b5-ad3f1b3dad9f.png)

**Comment by anubhav06** [comment]
Yes, I tried my best to make the cart page similar to Swiggy. (Although I am not good with frontend designing 😅)
![0](https://user-images.githubusercontent.com/43821031/154996447-60315aa1-297c-47c1-8f46-78cfe16d898c.png)

**Comment by anubhav06** [comment]
Yes! I tried my best to make the cart page similar to Swiggy (although I am not good with frontend designing 😅)
![0](https://user-images.githubusercontent.com/43821031/154996943-7cbe00d9-b01d-4985-aa58-825aa683f801.png)

**Comment by KendallDoesCoding** [comment]
> Yes! I tried my best to make the cart page similar to Swiggy (although I am not good with frontend designing 😅) ![0](https://user-images.githubusercontent.com/43821031/154996943-7cbe00d9-b01d-4985-aa58-825aa683f801.png)

Yeah thought so! I mean it looks pretty good on your site too, but maybe like Swiggy it can show some icons.. Add a "Apply Coupon" field and show the restaurant name?

**Comment by anubhav06** [comment]
Yes definitely. 
I had planned to add the restaurant information as well (like Swiggy) but I think I dropped that idea because of the CSS work (as I am not good at CSS) 😂. But now that you have come up with this topic, so it can be opened for the community to contribute, if anyone wants to work on it.

As far as the "Apply Coupons" field is concerned, this will also require some backend work for the coupons to work (by validating the coupon codes). So I think, this can be opened as a separate new issue

**Comment by KendallDoesCoding** [comment]
> Yes definitely. I had planned to add the restaurant information as well (like Swiggy) but I think I dropped that idea because of the CSS work (as I am not good at CSS) 😂. But now that you have come up with this topic, so it can be opened for the community to contribute, if anyone wants to work on it.
> 
> As far as the "Apply Coupons" field is concerned, this will also require some backend work for the coupons to work (by validating the coupon codes). So I think, this can be opened as a separate new issue

Oh ok, you can make a branch and I'll try working on the CSS coz I'm decent at CSS

**Comment by anubhav06** [comment]
Pls check the [installation](https://github.com/anubhav06/eatit#installation-%EF%B8%8F) and [contributing](https://github.com/anubhav06/eatit/blob/main/.github/CONTRIBUTING.md) guidelines if you would like to contribute.

Also, just a clarification that for running the project, one needs to run the backend server as well (Django), since you mentioned that your computer cannot handle Django
